README.md

```diff
@@ -66,6 +67,29 @@ Then to install buildah follow the steps in this example:
   buildah --help
 ```
 
+In Ubuntu/Debian, you can use this command:
+
+```
+  apt-get -qq update
+  apt-get install software-properties-common
+  add-apt-repository ppa:duggan/bats
+  add-apt-repository ppa:alexlarsson/flatpak
+  add-apt-repository ppa:gophers/archive
+  apt-get install bats btrfs-tools git libapparmor-dev libdevmapper-dev libglib2.0-dev libgpgme11-dev libostree-dev libseccomp-dev libselinux1-dev
+  apt-get install golang-1.8
+```
+Then to install buildah on Ubuntu/Debian follow the steps in this example:
+
+```
+  mkdir ~/buildah
+  cd ~/buildah
+  export GOPATH=`pwd`
+  git clone https://github.com/projectatomic/buildah ./src/github.com/projectatomic/buildah
+  cd ./src/github.com/projectatomic/buildah
+  PATH=/usr/lib/go-1.8/bin:$PATH make runc all TAGS="apparmor seccomp"
```

Comment:
This make worked the first time through without an issue.  However if I tried to do make again, I got an error that it couldn't find files.  I'd to move the contents of the ~/buildah directory to ~/go and then it worked fine.  I'm not sure why that is....

Reply:
I always place my source under the GOPATH, then symbolic link to top level directory.

Reply:
ok, maybe it's an ubuntu thing, haven't run into this on other platforms.